client/src/ui/css/jsonview.css

```diff
@@ -0,0 +1,77 @@
+.line {
```

Comment:
Use 4 space indentation to format the code

Reply:
Fixed with https://github.com/wso2-extensions/identity-tools-plugin-vscode/pull/7